package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@heroicons/react": "^2.1.5",
```

Comment:
We could also use fa icon flag (https://fontawesome.com/icons/flag?f=classic&s=solid) and then we would not need this dependency.